labware-library/src/labware-creator/components/ConditionalLabwareRender.tsx

```diff
@@ -1,4 +1,5 @@
 import { useLayoutEffect, useRef, useState } from 'react'
+import { isEqual } from 'lodash'
```

Comment:
nit - this way we don't import the whole `lodash` package and instead only load the specific fn we want

```suggestion
import isEqual  from 'lodash/isEqual'
```